src/scripts/main.js

```diff
@@ -1,5 +1,34 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+wall.addEventListener('click', ev => {
+  let xPosition
+    = ev.clientX
+    - wall.getBoundingClientRect().left
+    - (spider.clientWidth / 2);
+  let yPosition
+    = ev.clientY
+    - wall.getBoundingClientRect().top
+    - (spider.clientHeight / 2);
```

Comment:
не найкращий запис, можна винести  wall.getBoundingClientRect().left і top в змінні, щоб рядки не були такі довгі
```suggestion
  let xPosition = ev.clientX - wall.getBoundingClientRect().left
    - (spider.clientWidth / 2);
  let yPosition = ev.clientY - wall.getBoundingClientRect().top
    - (spider.clientHeight / 2);
```